turbine/src/xdp.rs

```diff
@@ -201,9 +220,12 @@ impl XdpRetransmitter {
                             None,
                             src_ip,
                             src_port,
-                            None,
                             receiver,
                             drop_sender,
+                            move |ip| {
+                                let r = atomic_router.load();
+                                r.route(*ip).ok()
```

Comment:
one thing to note is that we now do a load for each route lookup here which is
not ideal. But we're close to getting rid of tx_loop entirely as API soon, so
it's fine